docs/_data/components/schemas/headers.yaml

```diff
@@ -98,4 +95,52 @@ real-time-snapshot-header:
   required:
     - last-updated
     - snapshot-provenance
-    - power-system-resources    
+    - power-system-resources
+
+seasonal-proposal-header:
+  <<: *common-header
+  description: |
+
+    Describes the contents of this seasonal ratings proposal.
+
+
+seasonal-proposal-header-slim:
+  <<: *common-header
+  description: |
+
+    Describes the contents of the seasonal proposal. In particular, the
+    structure of the `ratings` is determined by the details provided in this
+    header.
+
+  properties:
+    <<: *common-header-props
+    power-system-resources:
+      $ref: ./array-max-monitored-elements.yaml#/ordered-named-power-system-resources
+    default-seasonal-schedule:
+      description: |
+
+        Defines the seasonal schedule that is used in the proposal.  Each season
+        is defined in the `schedule` array and the `ends` property defines the
+        end of the last season.
+
+        Each individual facility proposal must have 
+
+      properties:
+        schedule:
+          $ref: ./array-max-seasons.yaml#/seasonal-schedule
+        ends:
+          $ref: ./period-start.yaml
+        day-night:
+          $ref: ./day-night.yaml
+      required:
+        - schedule
+        - ends
+    limit-type:
```

Comment:
@getorymckeag Instead of doing this, let's parameterize the "slim" media type name, i.e., `application/vnd.trolie.seasonal-ratings-proposal-slim.v1+json; limit-type=apparent-power`

This leverages the media type standard to be explicit about the content without introducing a "slim" media type for each limit type. It also avoids the awkward explanations about why this requires a value that contributes nothing semantically.

The standard could define `Content-Type: application/json` in the `PATCH` to be interpreted at runtime by the Clearinghouse Provider as the "slim" media type with a fixed `limit-type=` parameter. The response would always have an explicit media type, of course, even when the client does a `GET Accept: */*, application/json`.